source/whats-new.txt

```diff
@@ -51,12 +51,16 @@ Learn what's new in:
 What's New in 2.0
 -----------------
 
-The 2.0 {+driver-short+} release supports various new features, provides
-a smoother timeout experience, and implements clearer, more idiomatic Go
+The 2.0 {+driver-short+} release supports OpenID Connect (OIDC)
```

Comment:
This looks good in general, but let me just figure out if we're ready to advertise OIDC